compiler/analyzer.ml

```diff
@@ -30,6 +30,37 @@ let builtins = VarMap.add "print" {
   name = "print";
   s_type = Func({ param_types = [Any]; return_type = Void; });
 } builtins
+let builtins = VarMap.add "uniform" {
```

Comment:
this should be removed, it won't be a builtin as it will be included in the dist standard library (see #107)